app/directives/statsTable.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+angular
+  .module('quickGoFeApp')
+  .directive('statsTable', [ function() {
+    return {
+      restrict: 'E',
+      templateUrl: 'directives/statsTable.html',
+      scope: {
+        'title': '@',
+        'items' : '=',
+        'isTerm' : '@?'
+      }, link: function(scope) {
+        scope.isTerm = scope.isTerm ? scope.isTerm : false;
+        scope.formatNumber = function(number) {
```

Comment:
This formatNumber does the same as prettyPrintNumberAnnotations in annotationList.js.
It does look like a function at a service level could work.

Of course we will have similar cases all over, but if we start bit by bit, then we will get a unified number formatting.